ddb-client/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     testImplementation testFixtures(project(':opensearch-remote-metadata-sdk-core'))
     implementation project(':opensearch-remote-metadata-sdk-remote-client')
     implementation project(':opensearch-remote-metadata-sdk-aos-client')
+    implementation group: 'software.amazon.awssdk', name: 'netty-nio-client', version: "2.30.18"
```

Comment:
we use the BOM, don't specify a version

(might need to move it down below line 13)